cv32/tests/cfg/no_pulp.yaml

```diff
@@ -0,0 +1,9 @@
+name: no_pulp
+description: Sets all PULP-related parameters to 0
+compile_flags: >
+    +define+NO_PULP
+    +define+HAHAHA
```

Comment:
You either left this here to see if I was watching, or there is an in-house SiLabs joke behind it.